lib/src/core/better_player_controller.dart

```diff
@@ -1329,6 +1348,14 @@ class BetterPlayerController {
     }
   }
 
+  void detachVideoPlayerController() {
+    if (videoPlayerController != null) {
+      videoPlayerController?.removeListener(_onFullScreenStateChanged);
+      videoPlayerController?.removeListener(_onVideoPlayerChanged);
+      videoPlayerController = null;
```

Comment:
```suggestion
      videoPlayerController.removeListener(_onFullScreenStateChanged);
      videoPlayerController.removeListener(_onVideoPlayerChanged);
      videoPlayerController = null;
```

Reply:
nó là biến toàn cục nên phải có check null như vậy, không là lỗi compile ngay anh ạ

Reply:
`videoPlayerController!.removeListener(_onFullScreenStateChanged);`

Reply:
như vậy với như em đang dùng thì có cải thiện hơn gì vậy anh?

Reply:
nó là biến toàn cục nên vẫn có thể bị set bằng null sau thời điểm đã check != null. Vậy nên IDE mới báo lỗi khi mà mình không dùng check null anh ạ